canvas_modules/harness/test_resources/parameterDefs/toggle_paramDef.json

```diff
@@ -0,0 +1,484 @@
+{
+  "titleDefinition": {
+    "title": "toggle Fields"
+  },
+  "current_parameters": {
+    "string": null,
+    "string_null": null,
+    "string_empty": "",
+     "string_error": null,
+    "string_warning":null,
+    "hide": true,
+    "string_hidden": true,
+    "disable": true,
+    "string_disabled": true,
+    "string_table": [
+      [
+        [
+          "Hopper"
+        ],
+        [
+          "Turing"
+        ],
+        [
+          "Shannon"
+        ],
+        [
+          "Babbage"
+        ]
+      ],
+      []
+    ],
+    "string_table_cond": [
+      [
+        [
+          "Hopper"
+        ],
+        [
+          "Turing"
+        ],
+        [
+          "Shannon"
+        ],
+        [
+          "Babbage"
+        ]
+      ],
+      []
+    ]
+  },
+  "parameters": [
+    {
+      "id": "string",
+      "type": "string"
```

Comment:
Since toggles are boolean we shouldn't be defining everything as `string`.  These should be boolean.  The names are also wrong.  I would probably just name them `toggle_...`